Firmware/util.cpp

```diff
@@ -181,32 +181,23 @@ inline int8_t is_provided_version_newer(const char *version_string)
     return 0;
 }
 
-bool force_selftest_if_fw_version()
+bool eeprom_fw_version_older_than(const uint16_t (&ver_req)[4])
```

Comment:
we can probably optimize the parameter as PROGMEM

Reply:
Done